src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -30,10 +32,11 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
     private var url: URL?
     private var touchHelperView: UIView?
     private var loadingView: LoadingView?
+    private var webView: WKWebView!
```

Comment:
This is still implicitly unwrapping `webView` and will cause a warning with our updated SwiftLint configuration soon. Please avoid force unwrapping wherever possible (this however is allowed for `IBOutlets`), there are still a lot of force unwrappings in your class.

Reply:
I have made changes, Please review it and let me know if I am still missing out anything.